js/letters.js

```diff
@@ -17,8 +17,24 @@ var counter = {
 
 function countLetters(counter, sample_text){
   // FIX ME
+  if (sample_text.length === 0){
+    return counter;
+  }
+  for (let property in counter){
+    if (property === sample_text.slice(-1).toLowerCase()){
```

Comment:
Instead of writing `sample_text.slice(-1).toLowerCase()`  twice, you can just place this into a variable